packages/web/scripts/workers-site/index.js

```diff
@@ -343,7 +343,8 @@ function isAssetUrl(url) {
     pathname.startsWith('/favicons') ||
     pathname.startsWith('/manifest.json') ||
     pathname.startsWith('/.well-known') ||
-    pathname.startsWith('/documents')
+    pathname.startsWith('/documents') ||
+    pathname.startsWith('/.version')
```

Comment:
nitty, but can we call this .gitsha or something like that? we have so much version overloading